scripts/generate-config-data.js

```diff
@@ -63,9 +66,16 @@ function parseDefaultValue (defaultValue) {
       return [];
     default:
       try {
+        // Remove thousands separators if numeric value
+        if (defaultValue.match(/^[\d_]+$/)) {
```

Comment:
This regexp matches on wacky strings like e.g. `________000__` – not sure if that's a big deal, but you might want to lock it down a little? Also, perhaps `defaultValue.test(……)` would work here? Or even skip the test entirely and just `replace()`?